src/Illuminate/Database/Query/Grammars/MySqlGrammar.php

```diff
@@ -94,6 +95,82 @@ protected function compileIndexHint(Builder $query, $indexHint)
         };
     }
 
+    /**
+     * Compile a group limit clause.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @return string
+     */
+    protected function compileGroupLimit(Builder $query)
+    {
+        return $this->useLegacyGroupLimit($query)
+            ? $this->compileLegacyGroupLimit($query)
+            : parent::compileGroupLimit($query);
+    }
+
+    /**
+     * Determine whether to use a legacy group limit clause for MySQL < 8.0.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @return bool
+     */
+    public function useLegacyGroupLimit(Builder $query)
+    {
+        $version = $query->getConnection()->getReadPdo()->getAttribute(PDO::ATTR_SERVER_VERSION);
+
+        return ! $query->getConnection()->isMaria() && version_compare($version, '8.0.11') < 0;
+    }
+
+    /**
+     * Compile a group limit clause for MySQL < 8.0.
+     *
+     * Derived from https://softonsofa.com/tweaking-eloquent-relations-how-to-get-n-related-models-per-parent/.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @return string
+     */
+    protected function compileLegacyGroupLimit(Builder $query)
+    {
+        $limit = (int) $query->groupLimit['value'];
+        $offset = $query->offset;
+
+        if (isset($offset)) {
+            $offset = (int) $offset;
+            $limit += $offset;
+
+            $query->offset = null;
+        }
+
+        $column = last(explode('.', $query->groupLimit['column']));
+        $column = $this->wrap($column);
```

Comment:
@staudenmeir hi. why remove the table's name?

```diff
-$column = last(explode('.', $query->groupLimit['column']));
-$column = $this->wrap($column);
+$column = $this->wrap($query->groupLimit['column']);
```

Look at this:
`Column not found: 1054 Unknown column 'auditable_id' in 'field list'`
`https://github.com/owen-it/laravel-auditing/issues/974`

is that related?

Reply:
Removing the table name is not the issue here. If the column isn't found without the table name, it can't be found it with the table name either.